client/src/App.js

```diff
@@ -4,16 +4,16 @@ import { BrowserRouter, Route } from "react-router-dom";
 
 import { theme } from "./themes/theme";
 import LandingPage from "./pages/Landing";
-import SimpleDialogDemo from "./pages/GmailAPI";
+import GmailDialog from "./pages/GmailDialog";
 
 import "./App.css";
 
 function App() {
   return (
     <MuiThemeProvider theme={theme}>
       <BrowserRouter>
-        <Route  exact path="/" component={LandingPage} />
-        <Route path="/gmail" component={SimpleDialogDemo} />
+        <Route exact path="/" component={LandingPage} />
+        <Route path="/gmail" component={GmailDialog} />
```

Comment:
Future consideration:
Btw, we can make this dialog appear above other things with a variable path using a path like this:
"*/gmail"